api/serializers.py

```diff
@@ -3,6 +3,8 @@
 from django_countries.serializers import CountryFieldMixin
 from django_countries.fields import CountryField
 from django_countries.fields import Country
+from brand.models.contact import Contact
+from django.contrib.auth.models import User
```

Comment:
`User` is no longer necessary to import. Neither is `CountryField`, incidentally.